terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfView.java

```diff
@@ -0,0 +1,115 @@
+        form.setField("tourName", downloadPDFOutput.getTourName());
+        form.setField("accomName", downloadPDFOutput.getAccomName());
+        form.setField("customerKana", downloadPDFOutput.getCustomerKana());
+        form.setField("customerTel", downloadPDFOutput.getCustomerTel());
+        form.setField("adultUnitPrice", String.valueOf(downloadPDFOutput
+                .getAdultUnitPrice()));
+        form.setField("reservedDay", String.valueOf(sdf.format(downloadPDFOutput
+                .getReservedDay())));
+        form.setField("conductor", downloadPDFOutput.getConductor());
+        form.setField("tourAbs", downloadPDFOutput.getTourAbs());
+        form.setField("customerAdd", downloadPDFOutput.getCustomerAdd());
+        form.setField("customerJob", downloadPDFOutput.getCustomerJob());
+        form.setField("tourDays", downloadPDFOutput.getTourDays());
+        form.setField("depDay", String.valueOf(sdf.format(downloadPDFOutput
+                .getDepDay())));
+        form.setField("customerName", downloadPDFOutput.getCustomerName());
+        form.setField("childUnitPrice", String.valueOf(downloadPDFOutput
+                .getChildUnitPrice()));
+        form.setField("depName", downloadPDFOutput.getDepName());
+        form.setField("customerBirth", String.valueOf(sdf.format(
+                downloadPDFOutput.getCustomerBirth())));
+        form.setField("arrName", downloadPDFOutput.getArrName());
+        form.setField("customerMail", downloadPDFOutput.getCustomerMail());
+        form.setField("adultCount", String.valueOf(downloadPDFOutput
+                .getAdultCount()));
+        form.setField("customerCode", downloadPDFOutput.getCustomerCode());
+        form.setField("reserveNo", downloadPDFOutput.getReserveNo());
+        form.setField("remarks", downloadPDFOutput.getRemarks());
+        form.setField("accomTel", downloadPDFOutput.getAccomTel());
+        form.setField("customerPost", downloadPDFOutput.getCustomerPost());
+        form.setField("printDay", String.valueOf(sdf.format(downloadPDFOutput
+                .getPrintDay())));
+        form.setField("adultPrice", String.valueOf(downloadPDFOutput
+                .getAdultPrice()));
+        form.setField("childPrice", String.valueOf(downloadPDFOutput
+                .getChildPrice()));
+        form.setField("sumPrice", String.valueOf(downloadPDFOutput
+                .getSumPrice()));
+        form.setField("paymentTimeLimit", downloadPDFOutput
+                .getPaymentTimeLimit());
+        stamper.setFormFlattening(true);
+        stamper.setFreeTextFlattening(true);
+        stamper.close();
+
+        response.setContentType("application/pdf");
+        response.setCharacterEncoding("UTF-8");
+        response.setHeader("Content-Disposition", "attachment; filename="
+                + downloadPDFOutput.getPdfName());
+    }
+
+    @Override
+    public String getUrl() {
+        return reservationReportPdfPath;
+    }
```

Comment:
- [ ] Good to rearrange methods. Criteria are access modifiers and logical order.

* public `getUrl`
* protected `mergePdfDocument`

Reply:
05f3b9c2740d4f78b9d5cd422422fbcbeb20d29e

I rearranged method order in above commit .